src/Config.js

```diff
@@ -118,6 +119,43 @@ export class Config {
     this.validateSchemaOptions(schema);
     this.validateEnforcePrivateUsers(enforcePrivateUsers);
     this.validateRequestKeywordDenylist(requestKeywordDenylist);
+    this.validaterateLimit(rateLimit);
+  }
+
+  static validaterateLimit(rateLimit) {
+    if (!rateLimit) {
+      return;
+    }
+    if (typeof rateLimit !== 'object') {
```

Comment:
Object type needs to be validated like so:

```suggestion
    if (Object.prototype.toString.call(rateLimit) !== '[object Object]') {
```